### PR TITLE
WIP: Update ZigBee feature to split bundles per dongle

### PR DESCRIPTION
**Pull request by cdjackson**
As discussed [here](https://github.com/openhab/org.openhab.binding.zigbee/issues/47#issuecomment-366528001) the new configuration for ZigBee bundles requires the features to be split.

I'm not sure if there's additional changes required - I could guess that the mvn location for the library artefacts need to be defined somewhere. 

Please let me know if there's anything I need to add or change here. Once the structure is correct I'll also need to add the other dongles - at the moment I've just included a feature for the Ember dongle.

Clearly merging of this PR needs to be coordinated with [the PR to split the bundles](https://github.com/openhab/org.openhab.binding.zigbee/pull/159).

Signed-off-by: Chris Jackson <chris@cd-jackson.com>

### CODE REVIEW

File: features/addons/src/main/feature/feature.xml [hunk 1]
```diff
@@ -3,10 +3,13 @@
 
 	<!-- these are 2.x add-ons in separate repositories (which do not have their own Karaf feature defined), so we include them here -->
 
-    <feature name="openhab-binding-zigbee" description="ZigBee Binding" version="${project.version}">
+    <feature name=“openhab-binding-zigbee-ember" description="ZigBee Binding for Ember NCP" version="${project.version}">
```
Comment (maggu2810) [inline features/addons/src/main/feature/feature.xml]: With respect to the Github diff view I assume the quote character in front of openHAB is not the one that has been expected.

**Comment by cdjackson** [comment]
Well spotted - thanks. The damn MacOS Xcode editor does this :(

File: features/addons/src/main/feature/feature.xml [hunk 2]
```diff
@@ -3,10 +3,13 @@
 
 	<!-- these are 2.x add-ons in separate repositories (which do not have their own Karaf feature defined), so we include them here -->
 
-    <feature name="openhab-binding-zigbee" description="ZigBee Binding" version="${project.version}">
+    <feature name="openhab-binding-zigbee-ember" description="ZigBee Binding for Ember NCP" version="${project.version}">
```
Comment (kaikreuzer) [inline features/addons/src/main/feature/feature.xml]: Why did you rename this feature? [My suggestion](https://github.com/openhab/org.openhab.binding.zigbee/issues/47#issuecomment-366700799) was to leave things as they are in the face of the user and just add all the different bundles in this feature.

File: features/addons/src/main/feature/feature.xml [hunk 3]
```diff
@@ -3,10 +3,13 @@
...
         <feature>openhab-runtime-base</feature>
         <feature>openhab-transport-serial</feature>
         <bundle start-level="80">mvn:org.openhab.binding/org.openhab.binding.zigbee/${project.version}</bundle>
+        <bundle start-level="80">mvn:org.openhab.binding/org.openhab.binding.zigbee.ember/${project.version}</bundle>
+        <bundle start-level="80">mvn:com.zsmartsystems.zigbee/com.zsmartsystems.zigbee</bundle>
```
Comment (kaikreuzer) [inline features/addons/src/main/feature/feature.xml]: Hm, that won't really work - addons must not bring any external dependencies that are not part of the openhab base feature (or available in the TP of the IDE). It'll be better to continue to include them as a lib in the appropriate version, I guess.

File: features/addons/src/main/feature/feature.xml (hunk 2 above)
Comment (cdjackson) [reply features/addons/src/main/feature/feature.xml]: Ah - ok, I misunderstood. no problem, I can add all 4 - no problem.

File: features/addons/src/main/feature/feature.xml (hunk 3 above)
Comment (cdjackson) [reply features/addons/src/main/feature/feature.xml]: I thought as much, but hoped there might have been another setting somewhere to set the repo. I can look at adding the files to the JARs, or something similar, but I see you have other comments on the overall refactoring that we probably need to discuss further first ;)

**Comment by kaikreuzer** [comment]
Superseded by https://github.com/openhab/openhab-distro/pull/676.
